BenchmarkVCFs/FindSamplesAndBenchmark.wdl

```diff
@@ -14,15 +14,16 @@ workflow FindSamplesAndBenchmark {
         File? gatkJarForAnnotation
         Array[String]? annotationNames = []
 
-        File ref_fasta
-        File ref_fasta_index
-        File ref_fasta_dict
-
+        File ref_fasta = "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.fasta"
+        File ref_fasta_index = "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.fai"
+        File ref_fasta_dict = "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.dict"
         File haplotype_database = "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.haplotype_database.txt"
-        File picard_cloud_jar = "gs://broad-dsde-methods/picard/picardcloud-2.26.6.jar"
+        String referenceVersion = "1"
```

Comment:
I think this should be "hg38", per the documentation of BenchmarkVCF

Reply:
Good call